wcs-importer-exporter.php

```diff
@@ -50,6 +50,9 @@ public static function init() {
 	 */
 	public static function setup_importer() {
 
+		if ( class_exists( 'WC_Subscriptions' ) && version_compare( WC_Subscriptions::$version, '2.0', '>=' ) ) {
+			add_filter('woocommerce_subscription_get_' . 'last_payment' . '_date', array(WCS_Importer_Exporter::class, 'last_payment_calculation'), 10, 3);
```

Comment:
Instead of `WCS_Importer_Exporter::class` here, the `__CLASS__` constant should be used to conform to the rest of the codebase.